src/components/ChataTable/ChataTable.js

```diff
@@ -751,6 +746,11 @@ export default class ChataTable extends React.Component {
       this.tableParams = params
 
       const nextTableParamsFormatted = formatTableParams(params, this.props.columns)
+      try {
```

Comment:
which bug is this fixing? Just curious because I remember it being really intentional where this callback was placed inside this function and moving it might break some things